src/App.js

```diff
@@ -1,6 +1,5 @@
 import { Routes, Route } from "react-router-dom";
 import "./styles/app.scss";
-
 import Home from "./pages/Home";
```

Comment:
Probably you didn't handle those relative paths. Please make sure to avoid getting repeating reviews.